index.html

```diff
@@ -1642,6 +1642,15 @@ <h2 id="OfflineAudioContext">
             </dl>
           </dd>
           <dt>
+            readonly attribute unsigned long length
```

Comment:
I find `length` is more consistent with the current constructor pattern. We
already use the term `duration` to specify the number is in 'seconds'.

On Fri, Feb 12, 2016 at 12:31 PM rtoy <notifications@github.com> wrote:

> In index.html
> <https://github.com/WebAudio/web-audio-api/pull/691#discussion_r52794639>:
>
> > @@ -1642,6 +1642,15 @@ <h2 id="OfflineAudioContext">
> >              </dl>
> >            </dd>
> >            <dt>
> > +            readonly attribute unsigned long length
>
> Oops. It's numberOfFrames in chrome's idl. But numberOfFrames is probably
> more intuitive than just length.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/WebAudio/web-audio-api/pull/691/files#r52794639>.
>